content/posts/2019-08-03---Going-from-ember-to-vue.md

```diff
@@ -71,13 +75,19 @@ The JavaScript of a component is too tightly tied to the markup (as it should be
 ## State management
 
 The popular state management solution seems to be [Vuex](https://vuex.vuejs.org/) (another "official" Vue library written by core team members).
-This is much closer to [Redux](https://redux.js.org/) or [NGRX](https://ngrx.io/).
-I'm not going to claim that these patterns are the end-all-be-all solutions for state management on the front-end, but it's where most of the community seems to be going right now.
-Which is a huge plus over Ember.
-The skills/practices you pick up writing Vuex will translate to these other libraries.
+This is much closer to [Redux](https://redux.js.org/) or [NGRX](https://ngrx.io/) -- if you're familiar with those libraries, you'll be able to jump right into Vuex.
+I'm not going to claim that these libraries are the end-all-be-all solutions for state management on the front-end, but it's where most of the community seems to be going right now.
+Which I feel is a huge plus over Ember, since the skills/practices you pick up writing Vuex will translate over to React/Redux or something similar.
```

Comment:
state management in ember isn't specific to ember though -- can you explain this a bit more?